commit d2d844414fc6570bed5402bd7785132ca1a6fb50
Author: Carlos Garnacho <carlosg@gnome.org>
Date:   Wed Jun 28 23:41:21 2017 +0200

    build: Bump optional tracker dependency to 2.0
    
    No code changes required. It remains API compatible with the current
    usage in nautilus.

diff --git a/meson.build b/meson.build
index ef8293a53..85c3cb9da 100644
--- a/meson.build
+++ b/meson.build
@@ -82,7 +82,7 @@ if get_option ('enable-selinux')
 endif
 
 if get_option ('enable-tracker')
-    tracker_sparql = dependency ('tracker-sparql-1.0')
+    tracker_sparql = dependency ('tracker-sparql-2.0')
     conf.set10 ('ENABLE_TRACKER', true)
 endif
